test/lib/users/users_database_test.rb

```diff
@@ -21,66 +21,102 @@
 require "users/users_database"
 
 describe Users::UsersDatabase do
+  let(:databases) { described_class.all }
+  let(:fixture_root) { FIXTURES_PATH.join("root") }
+  let(:fixture_root2) { FIXTURES_PATH.join("root2") }
+  let(:fixture_root3) { FIXTURES_PATH.join("root3") }
+
+  before do
+    Users::UsersDatabase.all.clear
+
+    # Mock access time: files in root2 are more recent than files in root and root3
+    allow(File).to receive(:atime) do |path|
+      path =~ /root2/ ? Time.now : Time.now - 1200
```

Comment:
Same comment as before. In fact, with this code is not clear which is most recent, root or root3?